### PR TITLE
Estimation update.

### PR DESCRIPTION
**Pull request by shot0829**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

**Review by jpa198 — CHANGES_REQUESTED** [review]
What about train_model.py, model.py and all the data/config files in estimation? If anyone wants them they can time travel for them

File: estimation/estimate_wct.py [hunk 1]
```diff
@@ -89,29 +80,18 @@ def est_LF_chours_single(
         The number of cores to use, returns the argument n_cores
         if scale_ncores is not set. Otherwise returns the updated ncores.
     """
-    # Make a numpy array of the input data in the right shape.
     # The order of the features has to the same as for training!!
```
Comment (jasonmotha) [inline estimation/estimate_wct.py:83]: This comment probably needs an update.

File: estimation/estimate_wct.py [hunk 2]
```diff
@@ -140,20 +120,29 @@ def estimate_LF_chours(
...
-        ((data[:, 0] * data[:, 1] * data[:, 2])[:, None], data[:, -2][:, None]), axis=1
-    )
-
-    core_hours = estimate(
-        data,
-        model,
-        model_type,
-        platform_config[const.PLATFORM_CONFIG.LF_DEFAULT_NCORES.name],
-        lf_svr_input_data=svr_data,
+    if data.shape[1] != 6:
```
Comment (jasonmotha) [inline estimation/estimate_wct.py:123]: Update the docstring accordingly.

File: estimation/estimate_wct.py [hunk 3]
```diff
@@ -140,20 +120,29 @@ def estimate_LF_chours(
...
+        raise Exception("Invalid input data, has to 6 columns. One for each feature.")
+    #
+    coefficients = {
+        "a": 0.804_038_96,
```
Comment (jasonmotha) [inline estimation/estimate_wct.py:127]: I think something is up with your coefficients?

**Review by jasonmotha — CHANGES_REQUESTED** [review]

**Comment by jasonmotha** [comment]
As per, this will need a wiki page showing actual CH use vs old estimation and current estimation. Or verification against Ethan's plots showing the same results.

File: estimation/estimate_wct.py (hunk 3 above)
Comment (jpa198) [reply estimation/estimate_wct.py:127]: Underscores are valid in numbers, the expected use is like commas when writing large numbers. It helps keep track of where in the number you are :P
TL;DR unless the number itself is wrong, this is fine

Comment (jasonmotha) [reply estimation/estimate_wct.py:127]: Numbers are awfully precise. But if that's what the Dr ordered...

File: estimation/estimate_wct.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 """
-Functions for easy estimation of WC, uses pre-trained models.
+Functions for easy estimation of WCT, uses fomulas based on regression of data.
```
Comment (jasonmotha) [inline estimation/estimate_wct.py:3]: formulas

Comment (jasonmotha) [reply estimation/estimate_wct.py:3]: <strike> of data </strike>

**Comment by shot0829** [comment]
https://wiki.canterbury.ac.nz/display/QuakeCore/Validation+on+Estimation+changes

**Review by jpa198 — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]
